File: python/problem35.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        length = len(nums);
        if target <= nums[0]:
            return 0;
        if target > nums[-1]:
            return length;
        for i in range(length):
            if nums[i] >= target:
                return i;

File: Java/problem35.java
public class Solution
{
    public int searchInsert(int[] nums, int target) 
    {
        int length = nums.length;
        if(target <= nums[0])
            return 0;
        
        for(int i = 0; i < length; ++i)
        {
            if(nums[i] >= target)
                return i;
        }
        return length;
    }
}